lib/express/spcp.js

```diff
@@ -18,7 +19,7 @@ const LOGIN_TEMPLATE = fs.readFileSync(path.resolve(__dirname, '../../static/htm
 
 const MYINFO_ASSERT_ENDPOINT = '/consent/myinfo-com'
 
-function config (app, { showLoginPage, serviceProvider, idpConfig }) {
+function config(app, { showLoginPage, serviceProvider, idpConfig, cryptoConfig }) {
```

Comment:
Nitpick - mockpass eslint rules specify that function declarations have a space between function name and parameter list. Could you honour that please?